static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -41,63 +37,61 @@ function OnboardingViewTask({org}: Props) {
     .map(platform => onboardingState.platformToProjectIdMap[platform])
     .map(projectId => allProjects.find(p => p.slug === projectId))
     .filter(project => project && !project.firstEvent) as Project[];
+  if (projects.length === 0) {
```

Comment:
@Neo-Zhixing should we redirect if this happens?

Reply:
oh nvm this the sideabar

Reply:
no, it just means all projects are resolved so we hide this card. This is on the sidebar.